evap/evaluation/models.py

```diff
@@ -1992,15 +1992,7 @@ def send_participant_publish_notifications(cls, evaluations, template=None):
             template.send_to_user(participant, {}, body_params, use_cc=True)
 
     @classmethod
-    def send_textanswer_reminder(cls):
+    def send_textanswer_reminder_to_user(cls, user, evaluations):
```

Comment:
```suggestion
    def send_textanswer_reminder_to_user(cls, user: UserProfile, evaluations: List[Evaluation]):
        assert len(evaluations) != 0
```

(may require additional imports)